nix/pkgs/core.nix

```diff
@@ -29,7 +29,7 @@ rec {
 
     # run all the non-slow cargo tests
     cargo build --features 'build' -p holochain_wasm_test_utils
-    cargo test ''${CARGO_TEST_ARGS:-} --workspace --exclude holochain --exclude release-automation --lib --tests --profile fast-test -- --nocapture
+    cargo test ''${CARGO_TEST_ARGS:-} --workspace --features test_utils --exclude holochain --exclude release-automation --lib --tests --profile fast-test -- --nocapture
```

Comment:
This is what allowed tests to run again. @steveeJ @jost-s @thedavidmeister does this look ok?

Reply:
looks very ok to me

Reply:
Can you actually set feature flags on the workspace level like this?

Reply:
> Can you actually set feature flags on the workspace level like this?

yes, this was introduced with the [resolver v2](https://doc.rust-lang.org/cargo/reference/features.html#resolver-version-2-command-line-flags):

> With resolver = "2", the features flags allow enabling features for any of the packages selected on the command-line with -p and --workspace flags. ...